### ISSUE TITLE
A11y_VirtualAgentBots_VirtualAgentHome_ChatDialog_Keyboard: The scroll-able element that appears in the Chat section is not keybaord actionable.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/BotFramework-WebChat#2440]
**Environment Details:**
Browser Version: Microsoft Edge 44.18362.267.0
Windows Version: Windows10
Screen reader: Narrator, NVDA (2019.2)
MAS Violated: MAS 2.1.1

**Steps to Reproduce:**
1. Home page should open for Virtual Agent Bots.
2. Navigate to main content area chat dialog,  the left side of the screen.
3. Navigate to the "Chat" section and interact with the chat functionality provided.
4. A scroll-able element will appear then try to access it using a keyboard.

**Actual:**
When the keyboard user interacts within the Chat section a scroll-able element appears which is not accessible using a keyboard.

**Expected:**
The scroll-able element in the Chat section should be keyboard accessible so that the keyboard users are able to access it read the previous chat.

**User Impact:**
If the interactive elements are not keyboard accessible then the keyboard users won't be able to access the element and might miss out on some information.

**Comment by cwhitten** [comment]
Thank you the issue. However, we are in the midst of an internal a11y audit to meet WCAG 2.1. The mechanisms to open, track, and address issues uncovered during the audit is a different process. I will close this for now.
